draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -168,13 +168,10 @@ described below.
 
 4. Register the candidate's mDNS hostname as defined in {{RFC6762}}.
 
-5. If registering of the mDNS hostname fails, abort these steps. The candidate
-   is not exposed.
-
-6. Store the mDNS hostname and its related IP address in the ICE agent for
+5. Store the mDNS hostname and its related IP address in the ICE agent for
```

Comment:
Nit on the text: it feels something is missing for the case when the registration fails and one may wonder when reading these steps.

The other PR provides the necessary detail here (the name can be returned before the registration finishes), so we could mention here step 5 is not conditional on step 4 or is in parallel to 4, and also refer back to this step in the added paragraph below or in the text of the other PR.

Reply:
Agreed

Reply:
OK, I added some more words referring to step 5 and step 6 in below statement.